### PR TITLE
docs: option title for showSaveDialog is not supported for some platforms

### PR DESCRIPTION
**Pull request by Tabea4**
see #28718

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [ ] tests are [changed or added](https://github.com/electron/electron/blob/master/docs/development/testing.md)
- [x] relevant documentation is changed or added
- [ ] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: none

### CODE REVIEW

**Comment by miniak** [comment]
@Tabea4 instead of "some platforms will not show it" we should say which ones

File: docs/api/dialog.md [hunk 1]
```diff
@@ -185,7 +185,7 @@ The `filters` specifies an array of file types that can be displayed, see
 
 * `browserWindow` [BrowserWindow](browser-window.md) (optional)
 * `options` Object
-  * `title` String (optional)
+  * `title` String (optional) - Title of the dialog, some platforms will not show it.
```
Comment (codebytere) [inline docs/api/dialog.md:188]: This should specify the platforms on which it works. 

See
https://github.com/electron/electron/blob/2865d161b720e209f29ca84858492e095dc3e161/docs/api/dialog.md#L96-L103 for prior art.

**Review by codebytere — CHANGES_REQUESTED** [review]

**Comment by Tabea4** [comment]
@miniak @codebytere 
I think that this is not necessarily only a matter of the platform, but of the desktop environment. I’m on Ubuntu using the Gnome desktop environment. I don’t have the possibility to check this on every desktop environment, that’s why I just wanted to add a note, so that developers know that this might not work on their machine. This is similar to the option `title` of the `showMessageBoxSync` : https://www.electronjs.org/docs/api/dialog#dialogshowmessageboxsyncbrowserwindow-options, where it also only says `some platforms will not show it.`

**Review by zcbenz — COMMENTED** [review]
I think we could take this if it says "on Linux some desktop environments will not show it".

**Comment by Tabea4** [comment]
> I think we could take this if it says "on Linux some desktop environments will not show it".

I changed it accordingly.

**Review by zcbenz — APPROVED** [review]

File: docs/api/dialog.md [hunk 2]
```diff
@@ -154,7 +154,7 @@ dialog.showOpenDialog(mainWindow, {
 
 * `browserWindow` [BrowserWindow](browser-window.md) (optional)
 * `options` Object
-  * `title` String (optional)
+  * `title` String (optional) - Title of the dialog, on _Linux_ some desktop environments will not show it.
```
Comment (codebytere) [inline docs/api/dialog.md:157]: Let's clean this phrasing up a bit? 

> `title` String (optional) - The dialog title. Cannot be displayed on some Linux desktop environments.

File: docs/api/dialog.md [hunk 3]
```diff
@@ -185,7 +185,7 @@ The `filters` specifies an array of file types that can be displayed, see
...
```
Comment (codebytere) [inline docs/api/dialog.md:188]: Same as above.

File: docs/api/dialog.md (hunk 2 above)
Comment (Tabea4) [reply docs/api/dialog.md]: Changed it as suggested

File: docs/api/dialog.md (hunk 3 above)
Comment (Tabea4) [reply docs/api/dialog.md]: Changed it as well

**Review by codebytere — APPROVED** [review]

**Pull request merged**